<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>250</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>568</width>
      <height>172</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelLeft">
         <property name="text">
          <string>500</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>50</number>
         </property>
         <property name="maximum">
          <number>950</number>
         </property>
         <property name="singleStep">
          <number>50</number>
         </property>
         <property name="pageStep">
          <number>50</number>
         </property>
         <property name="value">
          <number>500</number>
         </property>
         <property name="tracking">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelRight">
         <property name="text">
          <string>500</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="radioButtonFull">
           <property name="toolTip">
            <string>1</string>
           </property>
           <property name="text">
            <string>Full Quality</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">modeButtonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelRight_7">
           <property name="text">
            <string>Resize</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QRadioButton" name="radioButtonLeftR">
             <property name="toolTip">
              <string>2</string>
             </property>
             <property name="text">
              <string>Left</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">modeButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonRightR">
             <property name="toolTip">
              <string>3</string>
             </property>
             <property name="text">
              <string>Right</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">modeButtonGroup</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QRadioButton" name="radioButtonHalf">
             <property name="toolTip">
              <string>2</string>
             </property>
             <property name="text">
              <string>1 / 2</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">sizeButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonQuarter">
             <property name="toolTip">
              <string>4</string>
             </property>
             <property name="text">
              <string>1 / 4</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">sizeButtonGroup</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="labelRight_6">
           <property name="text">
            <string>Blur</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QRadioButton" name="radioButtoLeftB">
             <property name="toolTip">
              <string>4</string>
             </property>
             <property name="text">
              <string>Left</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">modeButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonRightB">
             <property name="toolTip">
              <string>5</string>
             </property>
             <property name="text">
              <string>Right</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">modeButtonGroup</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="labelRight_2">
             <property name="text">
              <string>Size : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="sizeX">
             <property name="text">
              <string>15</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelRight_5">
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelRight_8">
             <property name="text">
              <string>15</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="labelRight_3">
               <property name="text">
                <string>stdX :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="stdX">
               <property name="text">
                <string>8</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="labelRight_4">
               <property name="text">
                <string>stdY :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="stdY">
               <property name="text">
                <string>8</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>sizeX</sender>
   <signal>textChanged(QString)</signal>
   <receiver>labelRight_8</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>572</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeX</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>572</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stdX</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>541</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stdY</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>612</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>625</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonQuarter</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonHalf</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonClicked()</slot>
  <slot>radioButtonClicked(bool)</slot>
 </slots>
 <buttongroups>
  <buttongroup name="sizeButtonGroup"/>
  <buttongroup name="modeButtonGroup"/>
 </buttongroups>
</ui>
